scripts/romanisim-make-stack

```diff
@@ -0,0 +1,267 @@
+                        nargs=1,
+                        help=(
+                            'Previous simulated file in chronological order used for persistence modeling.'))
+
+    # Random number seed
+    parser.add_argument('-r', '--rng_seed',
+                        type=int,
+                        default=None,
+                        help='Random number seed (int)')
+
+    # SCA detector identifier number
+    parser.add_argument('-s','--sca',
+                        type=int,
+                        default=0,
+                        help='SCA to simulate')
+
+    # Use CRDS for distortion maps
+    parser.add_argument('-u', '--usecrds',
+                        action='store_true',
+                        help='Use CRDS for distortion map')
+
+    # Use CRDS stored PSFs
+    parser.add_argument('-w', '--webbpsf',
+                        action='store_true',
+                        help='Use webbpsf for PSFs.')
+
+
+
+
+    args = parser.parse_args()
+
+    log.info('Starting simulation...')
+    log.warning("romanisim is under active development.  Its output has "
+                "not been formally validated; only limited testing has been "
+                "performed.  For this reason, use of romanisim for "
+                "preparation of ROSES proposals is not advised.  Other "
+                "packages like galsim's roman package or STIPS may better "
+                "serve such purposes.")
+
+    # Set up metadata
+    metadata = deepcopy(default_parameters_dictionary)
+
+    # If creating a script, create the script file
+    if args.make_script:
+        spcript_file = open("romansim_stack.script", "w")
+
+        # Acquire cleaned options to pass to script call
+        options_dct = {}
+        for option, value in vars(args).items():
+            if args.debug:
+                print(f'option, value = {option, value}')
+            if option not in ('input_file_name','debug','make_script') and value is not None:
+                options_dct[option] = value
+
+
+
+    with open(args.input_file_name, newline='') as infile:
+        reader = csv.DictReader(infile, delimiter=',', skipinitialspace=True)
+        for entry in reader:
+            # Debug: print keys
+            if args.debug:
+                print(f'entry.keys = {entry.keys()}')
+
+            # Reset time offset
+            time_offset = 0 * u.s
+
+            # RA & Dec
+            coord = SkyCoord(ra=float(entry['ra']) * u.deg, dec=float(entry['dec']) * u.deg,
+                             frame='icrs')
+            wcs.fill_in_parameters(metadata, coord, boresight=args.boresight, roll_ref=float(entry['roll_angle']))
+
+            # Observational metadata
+            metadata['instrument']['optical_element'] = entry['optical_element']
+            metadata['exposure']['ma_table_number'] = int(entry['ma_table_number'])
+            ma_table = parameters.ma_table[ metadata['exposure']['ma_table_number']]
```

Comment:
For this kind of stuff, I think you're right that we should pull out common code in romanisim-make-image and here to avoid duplication.  I think that either means making a new file in the module that has some of these routines and romanisim-make-image a thinner wrapper, or giving romanisim-make-image a --obslist argument that triggers a loop like this one.